lib/registration/customer.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Registration.Customer do
+         def is_valid(customer_details) do
+              %{name: name, aadhar: aadhar} = customer_details
+
+                if  check_name(name) && check_aadhar(aadhar) == true do
+                  true
+                end
+         end
+
+        def check_name(name) do
+          if String.valid?(name) == true do
+            if String.length(name) >= 3 do
+              IO.inspect(name)
+              true
+            end
+          else
+          false
+        end
+      end
+
+      def check_aadhar(aadhar) do
+        if is_integer(aadhar) == true do
+          IO.inspect(aadhar)
+         count =
+          aadhar
+          |> Integer.digits
+          |> length
+
+          if count == 12 do
```

Comment:
use regex... for matching the conditions for name.

Reply:
used regex

Reply:
![image](https://user-images.githubusercontent.com/32371403/219647919-c7b132a2-1ba7-47e8-8b5c-aee87a95d716.png)